thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/template.hbs

```diff
@@ -0,0 +1,86 @@
+{{#if anomaly}}
+  {{#if isNotResolved}}
+    <div class="card-container card-container--box-shadow">
```

Comment:
I see this is a copy paste from the legacy code. FYI, almost all of the css class defined below are not accessible in the ember app

Reply:
fixed